portfolio/src/main/webapp/style.css

```diff
@@ -1,78 +1,198 @@
-*{margin:0px; padding:0px;}
+* {
+  margin: 0px;
+  padding: 0px;
+}
+
 /*
-Purpose of this line it allow the background to extend throughout the whole page
-When only applied this padding rule to the body there was still an issue with the margin around the top by default
+Purpose of this line it allows the background to extend throughout the whole page
+When only applied this padding rule to the body there was still an issue with
+the margin around the top by default.
```

Comment:
I think this can be simplified without losing meaning: "This allows the background to extend throughout the whole page." Also, you can remove the blank line in between the comment and the code, to make it clear which code you're referring to.